index.html

```diff
@@ -270,76 +287,100 @@ <h2>
           Launching a Web App
         </h2>
         <p>
-          When launching a web app via any means (e.g. OS launch, protocol handling, file handling, share target, link capturing, etc.) the trigger of the launch should run the steps to [=launch a web app=] with a {{LaunchParams}} containing relevant launch information (e.g. share data in the event of a share target launch).
+          When launching a web app via any means the trigger of the launch
+          should run the steps to [=launch a web app=] with a {{LaunchParams}}
+          containing relevant launch information (e.g. share data in the event
+          of a share target launch).
+        </p>
+        <p class="note">
+          Examples of different web app launch triggers:
+          <ul class="note">
```

Comment:
I think `class="note"` here is causing it to split it into a separate box. Try removing that and it should properly nest inside the one box.

Reply:
Unfortunately note boxes are real finicky, the ul on its own gets split out of the note box.
Made the split out explicit and added a comment.